src/projen/datastore.ts

```diff
@@ -10,9 +10,15 @@ export interface DatastoreOptions {
 
 export class Datastore extends pj.Component {
 
+  protected readonly definitionFile: string;
```

Comment:
Why is this needed? You could use `this.options.definitionFile` couldn't you?

Reply:
After a night's sleep I don't know why I did that in the first place. Gonna change it.